pkg/handler3/handler.go

```diff
@@ -0,0 +1,248 @@
+/*
```

Comment:
I think that's fine for now since this is alpha and we're not going to cache the OpenAPI unless the feature is enabled (right?!), but this absolutely needs to use @DangerOnTheRanger's cache fix for beta! thanks.

Reply:
Definitely agree. I plan to gather some performance numbers for beta with the cache as well